### DISCUSSION TITLE
<title>Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by Emon-Rana** [community/community#43679] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I am a verified GitHub Developer Pack Owner. I was using GitHub Copilot, but it's not connecting to the server today. Showing This error.
![image](https://user-images.githubusercontent.com/34301572/211242594-67706927-8869-48e0-928f-3a0462009491.png)

![image](https://user-images.githubusercontent.com/34301572/211242711-9d31457d-dec4-4bd3-8b38-4dd7a6630b1e.png)

When I am Trying to Sign up It's showing a Subscription page Where as I am still a verified Student in GitHub Developer Pack.

![image](https://user-images.githubusercontent.com/34301572/211243103-716d2fbc-c381-4e7f-8a46-49972231bae6.png)

**Comment by otkrickey** [comment]
Solved: #43634
